kotlin/set-min-max-scale/README.md

```diff
@@ -1,23 +1,30 @@
-# Min max scale
+# Set min max scale
 
-Restrict zooming to a specific scale range.
+Restrict zooming between specific scale ranges.
 
-![Min max scale](set-min-max-scale.png)
+![Image of min max scale](set-min-max-scale.png)
 
-## How it works
+## Use case
+
+Data may only appear at a certain scale on a map, and may be visually lost if zooming too far in or out. Setting the minimum and maximum scales ensures the zoom extents are appropriately limited for the purposes of the map.
+
+## How to use the sample
 
-1. Create an ArcGIS map.
-1. Set min and max scales of map, `ArcGISMap.maxScale = ...` and `ArcGISMap.minScale = ...`.
-1. Set initial Viewpoint of map, `ArcGISMap.initialViewpoint = ...`
-1. Set the ArcGIS map to the `MapView`.
+Zoom in and out of the map. The zoom extents of the map are limited between the given minimum and maximum scales.
+
+## How it works
 
+1. Instantiate an `ArcGISMap` object.
+2. Set min and max scales of map with `arcGisMap.minScale` and `arcGisMap.maxScale`.
```

Comment:
more common to just refer to the ArcGISMap as "map" for variable use here, or you can use ArcGISMap if you'd prefer.